### ISSUE TITLE
Linked Guide offline

### ISSUE THREAD

**Issue by besendorf** [Team-Neptune/DeepSea#132]
The linked guide is offline. 
https://switch.homebrew.guide/

**Comment by Slluxx** [comment]
Yeah, we know. We are still discussing either linking to other guides or making our own (since we dislike almost any other guide for one or another reason).

**Comment by Lotharl** [comment]
Looking forward to your guide, any idea when it will be up and the latest versions of software will be released?

**Comment by hax4dazy** [comment]
Soon™️

**Comment by marct3797** [comment]
Hey Guys, is there any new update for the community?
Thank you very much!
